documentation/library_guide/macros.rst

```diff
@@ -63,12 +63,35 @@ Macro                              Description
-                                   presence of the |onetbb_short| or |tbb_short| library. When the macro is not defined (by default)
-                                   or evaluates to a non-zero value,
-                                   parallel policies are executed using the |onetbb_short| or |tbb_short| library.
+                                   |tbb_long| for parallel policies (``par`` and ``par_unseq``)
+                                   When the macro is set to 0, algorithms with parallel policies do not
+                                   use this backend and try to find the "next" backend according to the priority.
+                                   This is recommended for code that should not depend on the
+                                   presence of the |onetbb_short| or |tbb_short| library.
+                                   When the macro evaluates to a non-zero value, algorithms with parallel policies
+                                   are executed using the |onetbb_short| or |tbb_short| library.
+                                   When the macro is not defined (by default) algorithms with parallel policies
+                                   are executed using the |onetbb_short| or |tbb_short| library unless another
+                                   parallel backend is explicitly enabled.
+
+                                   *TBB backend* has higher priority than *OpenMP backend*
+
+                                   Please note that if all parallel backend macros are set to 0, algorithms with
+                                   parallel policies are executed by the calling thread.
+---------------------------------- ------------------------------
+``ONEDPL_USE_OPENMP_BACKEND``      This macro controls the use of OpenMP* for parallel policies (``par`` and ``par_unseq``)
+                                   When the macro is set to 0, algorithms with parallel policies do not
+                                   use this backend and try to find the "next" backend according to the priority.
+                                   When the macro evaluates to a non-zero value the algorithms with parallel policies
+                                   are executed using OpenMP unless the backend with higher priority is explicitly enabled.
+                                   When this macro is not defined (by default) the algorithms with parallel policies
+                                   are executed on OpenMP if dedicated compiler option enables OpenMP (e.g. ``-fopenmp``) and
+                                   the backend with higher priority is explictly disabled.
+
+                                   *OpenMP backend* has lower priority than *TBB backend*.
+
+                                   Please note that if all parallel backend macros are set to 0, algorithms with
+                                   parallel policies are executed by the calling thread.
```

Comment:
I'd rewrite the description as follows:

This macro controls the use of OpenMP* for parallel execution policies (``par`` and ``par_unseq``). When the macro evaluates to a non-zero value, algorithms with parallel policies are executed using OpenMP unless the TBB backend is explicitly enabled (that is, the TBB backend takes precedence over the OpenMP backend). When the macro is not defined (by default) and no other parallel backends are chosen, a dedicated compiler option to enable OpenMP (such as ``-fopenmp``) also enables its use for algorithms with parallel policies. Setting the macro to 0 prevents use of OpenMP for parallel execution. If all parallel backends are disabled by setting respective macros to 0, algorithms with parallel policies are executed sequentially by the calling thread.

Reply:
done